ios-xcframework/Config/Gutenberg-Release.xcconfig

```diff
@@ -0,0 +1,17 @@
+#include "Gutenberg-Shared.xcconfig"
+
+DEBUG_INFORMATION_FORMAT = dwarf-with-dsym
+ENABLE_NS_ASSERTIONS = NO
+MTL_ENABLE_DEBUG_INFO = NO
+// Swift Compilation Mode
+//
+// This setting controls the way the Swift files in a module are rebuilt.\n- Incremental:
+// Only rebuild the Swift source files in the module that are out of date, running
+// multiple compiler processes as needed.\n- Whole Module: Always rebuild all Swift
+// source files in the module, in a single compiler process.
```

Comment:
Nitpick: I see `\n` characters in the comment, could we remove them and add new lines instead?